src/t_zset.c

```diff
@@ -72,12 +72,41 @@ void zsetConvertAndExpand(robj *zobj, int encoding, unsigned long cap);
 zskiplistNode *zslGetElementByRankFromNode(zskiplistNode *start_node, int start_level, unsigned long rank);
 zskiplistNode *zslGetElementByRank(zskiplist *zsl, unsigned long rank);
 
+static inline unsigned long zslGetNodeSpanAtHeight(zskiplistNode *x, int height) {
```

Comment:
There is another word "level", used in the structs. Let's distinguish between level and height?

0 ≤ *level* < *height* ?

```suggestion
static inline unsigned long zslGetNodeSpanAtLevel(zskiplistNode *x, int level) {
```